pallets/pallet-bonded-coins/src/types.rs

```diff
@@ -98,3 +98,10 @@ pub struct TokenMeta<Balance, Symbol, Name> {
 	pub symbol: Symbol,
 	pub min_balance: Balance,
 }
+
+#[derive(Debug, Encode, Decode, Clone, PartialEq, TypeInfo)]
+pub struct Team<AccountId> {
```

Comment:
```suggestion
pub struct PoolManagingTeam<AccountId> {
```

or something along those lines. Please use meaningful names for types and variables.

Reply:
Renamed in: [8d203c9](https://github.com/KILTprotocol/kilt-node/pull/768/commits/8d203c9413603fb8da1dbddbf75e117eda1d7b51)